.github/workflows/lint-test.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
           python-version: '3.10'
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.4.0
+        uses: helm/chart-testing-action@v2.6.0
```

Comment:
These were already changed in https://github.com/coredns/helm/pull/146, please remove the changes to the workflow files